pkg/cluster/internal/create/kubeadm-join.go

```diff
@@ -142,6 +148,8 @@ func runKubeadmJoinControlPlane(ec *execContext, configNode *NodeReplica) error
 		// preflight errors are expected, in particular for swap being enabled
 		// TODO(bentheelder): limit the set of acceptable errors
 		"--ignore-preflight-errors=all",
+		// use advertise address to avoid issues with Ipv6
+		"--apiserver-advertise-address", advertisedAddress,
```

Comment:
Seems that this option is not available in v1.11 and this causes the gate job failure.
Appreciate any help/idea to overcome this, are there any alternatives in `kubeadm` version 1.11 or do we simply can skip this option and IPv6 in v1.11? @neolit123

Reply:
1.11 is going outside of the support skew with the release of 1.14 (late march), so +1.

Reply:
:thinking: seems that v1.11 is going away soon https://github.com/kubernetes/test-infra/pull/11476/files